### DISCUSSION TITLE
Compile issue

### DISCUSSION THREAD

**Discussion by mangerij** [idaholab/moose#20366] [Q&A General] [answered]
I routinely updated conda and pulled new MOOSE and now I am getting the following error:

`/home/LIST/mangeri/anaconda3/envs/moose/bin/../lib/gcc/x86_64-conda-linux-gnu/9.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libtimpi_opt.so.6, needed by /home/LIST/mangeri/projects/moose/framework/contrib/pcre/libpcre-opt.so, not found (try using -rpath or -rpath-link)`

What could this be from?

thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

Just a leftover compiled object.
try `make clobberall` and if it doesnt work, commit all your work then run `git clean -xfd` in the root directory of MOOSE

Guillaume

**Reply by mangerij** [reply]
Thanks, the second command worked to solve this issue.
